### DISCUSSION TITLE
Mobx-based web framework?

### DISCUSSION THREAD

**Discussion by TheYarin** [mobxjs/mobx#3538] [Ideas]
I can't help but feel like despite the effort put into making Mobx feel natural in React, those two frameworks have a different approach as to how state should be managed. And honestly, I prefer Mobx's approach.
Have you ever considered creating a web framework where Mobx is a first-class citizen?
I believe the minds behind Mobx's amazing, thoughtful developer experience can create a web framework that can truly rise above today's popular web frameworks.

**Comment by kubk** [comment]
You may be interested in this experiment from Solid.js author: https://github.com/ryansolid/mobx-jsx

As far as I know there are no plans to create a mobx-based SPA framework. To be honest I also don't like the direction React is going to right now, but I don't have qualifications or time to create an alternative.

The opinion is solely mine, it doesn't represent any Mobx official view on React's direction
